config/development.toml

```diff
@@ -1,7 +1,7 @@
 [log.console]
 enabled = true
 level = "DEBUG"
-log_format = "default"
+log_format = "json"
```

Comment:
Nit: You can revert this config change, since human readable logging is acceptable for development purposes.